RELEASES.md

```diff
@@ -1,3 +1,353 @@
+Version 1.7.0 (2016-03-03)
```

Comment:
Might be a good idea to make it more obvious this hasn't been released yet and that this is technically a draft?